llvm/test/Transforms/IndVarSimplify/indvars-preversing-debugloc.ll

```diff
@@ -0,0 +1,65 @@
+; RUN: opt < %s -passes=indvars -S | FileCheck %s
```

Comment:
Nit: Typo in file name: "preversing", should be "preserving"

Reply:
Ok, I corrected the file name 😂